bash_scripts/tests.sh

```diff
@@ -46,7 +46,7 @@ apt-get -y install git
 apt-get install swig -y
 
 # Install depedencies
-pip install .[tf,envs,reverb,testing_formatting,record_episode]
+pip install .[jax,envs,reverb,testing_formatting,record_episode]
```

Comment:
as you mentioned, we should delete jax_tests.sh

Reply:
Sorted. :call_me_hand: